packages/cli/src/sso.ee/saml/saml-helpers.ts

```diff
@@ -32,6 +33,10 @@ export function getSamlLoginLabel(): string {
 	return config.getEnv(SAML_LOGIN_LABEL);
 }
 
+export function isOidcLoginEnabled(): boolean {
```

Comment:
Does it make sense to have a function called `isOidcLoginEnabled` is the saml-helper file ?